### PR TITLE
Optimize save registry cache.

### PR DESCRIPTION
**Pull request by bxy4543**
Optimize save registry cache.
Change log file to /var/lib/sealer/log, because github action no permission to save logs when run e2e test.

### CODE REVIEW

File: utils/ssh/ssh.go [hunk 1]
```diff
@@ -80,40 +80,47 @@ type Client struct {
...
-	}
-
-	if host == "" {
-		return nil, fmt.Errorf("get cluster EIP failed")
+	err := WaitSSHReady(sshClient, 6, ipList...)
+	if err != nil {
+		return nil, err
 	}
-	err := WaitSSHReady(sshClient, host)
+	uLimit := `if [ "$(ulimit -n)" -lt "65535" ];then echo "* - nofile 65535" >> /etc/security/limits.conf; fi`
```
Comment (fanux) [inline utils/ssh/ssh.go:101]: Add ulimit in NewSSHClient is very not friendly, The best way to config ulimit is using plugin.

Comment (bxy4543) [reply utils/ssh/ssh.go]: > Add ulimit in NewSSHClient is very not friendly, The best way to config ulimit is using plugin.

well

**Review by kakaZhou719 — APPROVED** [review]

**Pull request merged**